### PR TITLE
Some bug fixes

### PR DESCRIPTION
**Pull request by bmaluenda**
-Removed path bug for Windows users.
-Removed bug for users that have a different local username than the
username they hold in the DB2 server (i.e. my local user is
'inodu_notebook_01', whereas my DB2 user is 'bmaluenda').
-Removed a comment that was of no use (leftover from some of my early
experimentations with this script).

### CODE REVIEW

File: switch_model/wecc/get_inputs.py [hunk 1]
```diff
@@ -17,14 +17,10 @@
 import os
 import sys
 import time
-
```
Comment (josiahjohnston) [inline switch_model/wecc/get_inputs.py]: This blank line denotes a separate between built-in python libraries and 3rd party python libraries that need to be installed. Common convention goes:
```
# built in libs
import os

# 3rd party libs
import sshtunnel

# modules from this project
import switch_model
```

**Review by josiahjohnston — COMMENTED** [review]
Looks fine to me other than deleting one intentionally blank line. Big-picture, I found the sshtunnel library to be an unstable pain in the ass that will freeze up badly if your code generates any errors (it starts a background process that does not exit cleanly if other parts of code raise an error). I ended up writing a more robust approach here: https://github.com/RAEL-Berkeley/wecc_data_migration/blob/master/db_utils.py but it probably doesn't work on Windows.

**Comment by rodrigomha** [comment]
I can confirm that the previous code for SSH tunneling was not working in Windows, and this one is working.

**Comment by bmaluenda** [comment]
Thanks for the comments. I thought this was the approach you had decided, so I tried to get it running for Windows too. I'm not so proficient with DB connections as to improve it on my own though...

**Comment by PatyHidalgo** [comment]
Thanks for working on this. We were only using Mac, so thank you!

**Comment by PatyHidalgo** [comment]
Hi Benjamin,

I was looking over this commit and you changed trans_fixed_*om*_fraction to
trans_fixed_*o_m*_fraction in get_inputs.py, however I believe the former
was the decided parameter to use (commit from josiah and I on Jun 14).


Josiah and Benjamin,
Is it ok if I change the param back to trans_fixed_*om*_fraction? I wonder
why Benjamin didn't get an error
from switch_model/transmission/transport/build.py

Thanks,

**Comment by bmaluenda** [comment]
I just noticed that the version of Switch I used for testing was the one at https://github.com/switch-model/switch (the original one) and not the switch-wecc one.

If you could change the parameter name back, that would be great. Sorry!

**Comment by PatyHidalgo** [comment]
Ok, no worries, I'll do that.





On Tue, Sep 12, 2017 at 3:18 PM, Benjamin Maluenda <notifications@github.com

**Pull request merged**